src/main/java/net/fabricmc/tinyremapper/IMappingProvider.java

```diff
@@ -39,5 +39,20 @@ public Member(String owner, String name, String desc) {
 		public String owner;
 		public String name;
 		public String desc;
+
+		@Override
+		public int hashCode() {
+			return (31 * (31 * owner.hashCode()) + name.hashCode()) + desc.hashCode();
+		}
+
+		@Override
+		public boolean equals(Object other) {
```

Comment:
Maybe it should start with `if (other == this) { return true; }` for performance for that case, though I don't know if that is needed